### PR TITLE
[FIX] Add dimensionality bug fix

### LINKED ISSUE #16: [BUG] Dimensions of masked data not considered for fit_transform
While working on the integration test, I saw there is a little bug in the code that probably comes from losing the context we had in tedana.

Basically, we are not considering that the data has been masked when giving `shape3d` to `fit_transform()`. We're currently giving the original dimensions when it should be the masked ones. 

Edit: The error comes from the fact that `apply_mask` from nilearn only keeps the values inside the mask, while `mapca` in tedana keeps everything.

### PR DESCRIPTION
**Pull request by eurunuela**
Closes #16 

This PR adds a missing line and replaces another one to ensure the matrix has the correct dimensions to perform the reshaping. I finally found out the issue was in [mapca.py line 149](https://github.com/ME-ICA/mapca/blob/58cbeea64bb3ed84c5bd0bf23807185f0125566b/mapca/mapca.py#L149).

### CODE REVIEW

**Comment by eurunuela** [comment]
Once we merge this PR I'll open one for integration tests that should help us advance with other open PRs.

**Comment by eurunuela** [comment]
This was closed by #26

**Comment by tsalo** [comment]
It looks like there are no changes associated with this PR now? Were they handled already in #26?

**Comment by tsalo** [comment]
Never mind!
